pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -111,6 +112,46 @@ func (s *Service) CreateInvitation() (*InvitationRequest, error) {
 	}, nil
 }
 
+//ReceiveInvitation receives invitation request
+//TODO to be implemented as part of #113
+func (s *Service) ReceiveInvitation(request *ReceiveInvitationRequest) (*ReceiveInvitationResponse, error) {
```

Comment:
This package should not accept/produce structures specific to the restapi. Furthermore, the Service already implements a `Handle` function - that should be the way to send DIDComm messages to it.

We discussed the didcomm service providing the ability for consumers to register pre/post event listeners - with post-event listeners it should be possible to build a `ReceiveInvitationResponse`

Reply:
This is not using REST API model, this model is specific to `pkg/client/didexchange`

Reply:
moved to `service.handle(didcommMsg)`